index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/graasp.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="Graasp Player | Digital Education Platform"
+      content="Knowledge sharing without borders. Communicate, Collaborate, Build Engaging Learning Experiences"
```

Comment:
Yes this is the entry file for vite, and since vite is used to bundle the tests (cypress-vite-dev-server) we need to have this file. The tests were easier to setup using vite than using webpack because of the use of path aliases, in webpack this requires a specific config and cypress setup to be taken into account, with vite the config is much easier and tests load super fast !